code/script.js

```diff
@@ -316,10 +437,25 @@ const checkMyGuess = (personToCheck) => {
   // 2. Set a Message to show in the win or lose section accordingly
   // 3. Show the win or lose section
   // 4. Hide the game board
+  if (personToCheck === secret.name) {
+    winOrLoseText.innerHTML = `Congratulation! ${personToCheck} is the one we selected for you! YOU WIN!!!`
+  } else {
+    winOrLoseText.innerHTML = `Sad... I'sorry but you filed, ${personToCheck} is incorrect. You should try ${secret.name}, play again?.`
+   }
+  
+  winOrLose.style.display = 'flex'
+  board.style.display = 'none' 
+  questionSection.style.display = 'none'
+  //board.classList.add('hide-board');
+  //winOrLose.classList.add('show-win-lose');
+  
 }
 
 // Invokes the start function when website is loaded
 start()
 
 // All the event listeners
 restartButton.addEventListener('click', start)
+questions.addEventListener('change',selectQuestion)
+filter.addEventListener('click',checkQuestion)
+playAgain.addEventListener('click',start)
```

Comment:
Super organised code. Clear and understandable. I would just recommend to add more  personal comments instead of the ones by technigo because it is for you.